### DISCUSSION TITLE
Why my built jar is 4.49MB, but official release jar is 38MB.

### DISCUSSION THREAD

**Discussion by EasonChow001** [microsoft/onnxruntime#7099] [Other Q&A] [answered]
I remove all reference to plugin spotless in file java/build.gradle due to some unknown reason,  which causes my not deploying proxy suitably for gradle. And I compile onnxruntime with command: 
`./build.sh --config Release --build_shared_lib --build_java --parallel`

And as seen, my build jar file is only 4.49MB, But the official release jar is about 38MB. How can appear this ? Is my jar built ok?

**Comment by EasonChow001** [comment, marked as answer]
@Craigacp Hi, bother u again.

**Reply by Craigacp** [reply]
The released jar has Windows, macOS and Linux binaries inside, yours will only have whatever platform you've compiled it on. The released one may also include debug symbols, I've not looked at the release pipeline for a while and I can't remember if it does.

**Reply by EasonChow001** [reply]
> @Craigacp Hi, bother u again.

well, i see.
